simphony_metadata.yml

```diff
@@ -16,7 +16,8 @@ CUDS_KEYS:
     parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:
+    data:
```

Comment:
Even though I'm in favor of defining attributes in metadata level and just describe them with extra metatada (e.g. name, type, default, etc.), but the current consensus so far has been having fixed metadata attributes as lower case and dynamic ones as upper case with CUBA prefix. Therefore I think CUBA.DATA is the way to go for now.